lib/main.js

```diff
@@ -123,7 +123,12 @@ export default {
         const showRule = atom.config.get('linter-eslint.showRuleIdInMessage')
 
         if (this.worker === null) {
-          return []
+          return Promise.resolve([{
```

Comment:
The specs expect a promise to be returned and were erroring without this.

Since this isn't within a `then` it is not automatically a promise.